tests/TestCase/Console/HelpFormatterTest.php

```diff
@@ -183,7 +183,7 @@ public function testHelpWithOptions(): void
         $parser->addOption('test', ['help' => 'A test option.'])
                ->addOption('number', [
                    'help' => 'The number',
-                   'default' => 2,
+                   'default' => '2',
```

Comment:
I wasn't sure about this one. Should we support setting an integer and cast to string? It's true that all values read in would be strings so they user shouldn't falsely expect an integer value.

@markstory

Reply:
Given that all values coming from command line will be strings I don't see any reason supporting integer here.

Reply:
I think only accepting strings makes sense.